src/main/groovy/de/dkfz/roddy/execution/cluster/pbs/PBSJobManager.groovy

```diff
@@ -688,4 +686,22 @@ class PBSJobManager extends ClusterJobManager<PBSCommand> {
         return PBSCommand.QSUB
     }
 
+
+    static JobState parseJobState(String stateString) {
```

Comment:
You could un-static this method and make it abstract. This way it would be enforced to implement it.